### PR TITLE
Multiple fields of study/semesters addEntryToMatriculationData

### PR DESCRIPTION
**Pull request by matthias-geuchen**
### Reason for this PR
- adding multiple fields of study/semesters atomically should be supported

### Changes in this PR
- change addEntryToMatriculationData transaction to take list of SubjectMatriculation objects

### CODE REVIEW

**Review by NikoBergemann — APPROVED** [review]

**Review by nklsd — APPROVED** [review]
looks good

**Review by sluis1 — APPROVED** [review]
nice!

**Review by david-buderus — APPROVED** [review]

**Review by lrschaef — COMMENTED** [review]
Note: I am not responsible for this API, you do not need to add every member of team lagom for review. Lars and/or David would be enough, because they are responsible for the HL Lagom API.

File: hyperledger_matriculation_api.md [hunk 1]
```diff
@@ -129,8 +129,7 @@ This method adds a single entry to the list of semesters in the MatriculationDat
 - ID = addEntryToMatriculationData
```
Comment (dhelmert) [inline hyperledger_matriculation_api.md]:
```suggestion
- ID = addEntriesToMatriculationData
```

Considering that we now add multiple entries, the transaction should be named accordingly

**Review by dhelmert — CHANGES_REQUESTED** [review]

**Review by ldklenner — APPROVED** [review]

**Review by tobias-wiese — APPROVED** [review]
Well, I did not look into the code as I'm a ship into holiday. I approve to allow the merge

**Comment by tobias-wiese** [comment]
Oh, did not see that @lrschaef . Thank you anyways 🙂

**Review by dhelmert — APPROVED** [review]

**Review by lrschaef — APPROVED** [review]

**Review by nklsd — APPROVED** [review]

**Review by bastihav — APPROVED** [review]
I agree with luca.

**Pull request merged**